flink-connectors/flink-connector-hive/src/test/resources/explain/testLoadData.out

```diff
@@ -0,0 +1,8 @@
+== Abstract Syntax Tree ==
+LOAD DATA LOCAL INPATH '$filepath' OVERWRITE INTO TABLE default.p_table PARTITION (dateint=2022)
+
+== Optimized Physical Plan ==
+LOAD DATA LOCAL INPATH '$filepath' OVERWRITE INTO TABLE default.p_table PARTITION (dateint=2022)
+
+== Optimized Execution Plan ==
+LOAD DATA LOCAL INPATH '$filepath' OVERWRITE INTO TABLE default.p_table PARTITION (dateint=2022)
```

Comment:
In Spark, the explain of load data still follows query explain format. So how about:

```
LoadData(filepath=[$filepath], table=[default.p_table], overwrite=[true], local=[true])
```

This looks more like a **plan**.